README.md

```diff
@@ -1,2 +1,626 @@
+| TDengineConfigParams.TD_STACTMODE                       |                                               boolean                                                | This parameter is used to set the reception of batch data                                                                                                                           | If set to True:< The source is TDengine Source, using SourceRecords \<type \>to create TDengine Sink object<br/>The source is TDengine CDC, using ConsumerRecords \<type \>to create TDengine Sink object | The type here is the type that receives data|
+| TDengineConfigParams.TD_SOURCETYPE                      |                                                string                                                | If the data is from a source, such as source or cdc                                                                                                                                 | TDengine source is set to "tdengine_stource", TDengine cdc is set to "tdengine_cdc"|
+| TDengineConfigParams.PROPERTY_KEY_MESSAGE_WAIT_TIMEOUT  |                                               integer                                                | Message timeout, in milliseconds, default value is 60000                                                                                                                            ||
+| TDengineConfigParams.PROPERTY_KEY_ENABLE_COMPRESSION    |                                               boolean                                                | Whether compression is enabled during the transmission process. true:  Enable, false:  Not enabled. Default is false                                                                |         |
+| TDengineConfigParams.PROPERTY_KEY_ENABLE_AUTO_RECONNECT | integer| to enable automatic reconnection or not. true:  Enable, false:  Not enabled. Default to false                                                                                       |                                                                                                                             |
+| TDengineConfigParams.PROPERTYKEY_RECONNECT_RETR_COUNT   |                                               integer                                                | number of automatic reconnection retries, default value 3                                                                                                                           | only takes effect when PROPERTYKEY-INABLE AUTO-RECONNECT is true|
+| TDengineConfigParams.PROPERTYKEYDISABLE_SSL_CERTVNet    |                                               boolean                                                | Disable SSL certificate verification. true:  close, false:  Not closed. The default is false                                                                                        ||
+
+#### Use Sink connector
+Write the received RowData type data into TDengine example:
```

Comment:
Use the same coding style for markdown, leave a blank line between Header and Detail.